src/lib.rs

```diff
@@ -437,13 +438,13 @@ pub fn main() {
         );
     }));
 
-    if utils::env::get_game_version() != semver::Version::new(13, 0, 1) {
-        skyline_web::DialogOk::ok(
-            "ARCropolis cannot currently run on a Smash version lower than 13.0.1<br/>Consider updating your game or uninstalling ARCropolis.",
-        );
-        // Do not perform any of the hook installation and let the game proceed as normal.
-        return;
-    }
+    // if utils::env::get_game_version() != semver::Version::new(13, 0, 1) {
```

Comment:
Given that some offsets have been reverted to being hardcoded for a specific version (as well as the hardcoded offsets for the patches in the fixes file), I'd say the check here should be enabled and set to 13.0.2 instead of 13.0.1, with the message specifying it can only run on 13.0.2.